README.md

```diff
@@ -85,6 +87,54 @@ Port-forward the chroma server service to localhost using the following command.
 kubectl port-forward service/chroma-service 8000:8000
 ```
 
+### Monitoring
+
+To run the monitoring service on k8s, `matcha provision` must be run beforehand. We will need to build and push the metric service application to ACR. This image will be used by Kubernetes deployment. Before that, we need to set two bash variables, one for ACR registry URI and another for ACR registry name. We will use matcha get command to do this.
+
+```bash
+acr_registry_uri=$(matcha get container-registry registry-url --output json | sed -n 's/.*"registry-url": "\(.*\)".*/\1/p')
+acr_registry_name=$(matcha get container-registry registry-name --output json | sed -n 's/.*"registry-name": "\(.*\)".*/\1/p')
+```
+
+Now we're ready to login into ACR, build and push the image to the ACR.
+
+```bash
+az acr login --name $acr_registry_name
+docker build -t $acr_registry_uri/monitoring:latest -f monitoring/metric_service/Dockerfile .
+docker push $acr_registry_uri/monitoring:latest
+```
+
+Line number 39 in [monitoring-deployment.yaml](./infrastructure/monitoring/monitoring-deployment.yaml#L39) should be updated to match the Docker image name which we've just pushed to the ACR, and it will need to be in the following format: `<name-of-acr-registry>.azurecr.io/monitoring`.
+
+Next, we apply the Kubernetes manifest to deploy the metric service and the metric database on AKS.
+
+```bash
+kubectl apply -f infrastructure/monitoring
+```
+
+Finally, once the pod is running, we verify that our monitoring service is working. The command below should provide an IP address for the metric service interface.
```

Comment:
Once `kubectl` has finished applying the manifest, we should verify that the monitoring service is running. Running the commands below will give you an IP address for the service, which we can then `curl` for a response: